[BUG] Machine stats are not showing if ssh doesn't have authentication

### Title

Machine stats are not showing if ssh doesn't have authentication

### Platform

N/A

### Server Installation Method

Docker

### Version

1.11.1

### Troubleshooting

- [x] I have examined logs and tried to find the issue
- [x] I have reviewed opened and closed issues
- [x] I have tried restarting the application
- [x] I have read Known Issues (https://github.com/Termix-SSH/Support/issues/296)

### The Problem

I'm connecting to my machine via Tailscale SSH and there's no authentication for this connection. I'm able to start ssh session and access the cli, but stats are not showing.

Logs:
```
Error: Too many authentication failures (4 attempts)
    at collectMetrics (file:///app/dist/backend/backend/ssh/server-stats.js:1330:15)
    at PollingManager.pollHostMetrics (file:///app/dist/backend/backend/ssh/server-stats.js:825:35)
[9:24:02 AM] [ERROR] [📊] Stats collector connection failed [op:stats_connect_failed,host:10]
```

### How to Reproduce

1. Add an SSH connection with Authentication: None
2. Check the logs or try to go to stats page

### Additional Context

_No response_

--- a/Casks/termix.rb
+++ b/Casks/termix.rb
@@ -1,6 +1,6 @@
 cask "termix" do
   version "2.0.0"
-  sha256 "a752ad4f05b4991b8a2ef986da80a86b79a70c93fe1045c4399fcf348a28c1d0"
+  sha256 "2e381ac504093bfa72d16ee20043640724394729b2ca0e6b26c9eac19aeb6d08"
 
   url "https://github.com/Termix-SSH/Termix/releases/download/release-#{version}-tag/termix_macos_universal_dmg.dmg"
   name "Termix"

--- a/src/backend/ssh/server-stats.ts
+++ b/src/backend/ssh/server-stats.ts
@@ -31,7 +31,9 @@ import { connectionPool, withConnection } from "./ssh-connection-pool.js";
 
 function supportsMetrics(host: SSHHostWithCredentials): boolean {
   const connectionType = host.connectionType || "ssh";
-  return connectionType === "ssh";
+  if (connectionType !== "ssh") return false;
+  if (host.authType === "none" || host.authType === "opkssh") return false;
+  return true;
 }
 
 function isTcpPingEnabled(statsConfig: StatsConfig): boolean {

